packages/search/src/common/create-filters.ts

```diff
@@ -0,0 +1,45 @@
+import { ISearchParams } from "./params";
+import { paramSchema } from "./param-schema";
+
+/**
+ * Create filters
+ *
+ * @export
```

Comment:
no need for a JSDoc `@export` declaration, TSDoc picks up on the keyword in the actual code.